src/main/java/JsonIO.java

```diff
@@ -0,0 +1,44 @@
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.reflect.TypeToken;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.PrintWriter;
+
+class JsonIO {
+    private Gson g;
+
+    JsonIO () {
+        g = new GsonBuilder().setDateFormat("yyyy-mm-dd").setPrettyPrinting().create();
+    }
```

Comment:
could have done a better name than the letter 'g'.

Reply:
Good point, modified to gson